src/styles/main.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables.scss";
```

Comment:
I don't know if it's mistake to import scss files after some properties but i recomend you write import first